code/src/components/EndOfGame.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import { useSelector } from 'react-redux'
+import "nes.css/css/nes.min.css"
+
+import { EndGamePageContainer, Header, RestartButtonContainer, Credits } from './styled-components/EndOfGameStyling'
```

Comment:
It is easier to read for everyone in this way:
```jsx
  import {
    EndGamePageContainer, 
    Header, 
    RestartButtonContainer, 
    Credits} from '../styling/style'
```

Reply:
If you use a unique username you should be able to play the game 👍 . It is not possible to use the same username twice. :)